app/forms/flood_risk_engine/steps/base_postcode_form.rb

```diff
@@ -0,0 +1,20 @@
+module FloodRiskEngine
+  module Steps
+    class BasePostcodeForm < BaseRedirectableForm
+
+      property :post_code, virtual: true
```

Comment:
this form has nothing to do with the address model, it's a virtual attribute that relates to the search parameter and `post_code` matches how params defined in the route. 

This form relates to collecting  a group of addresses, only one of which will ever be persisted 

 for sake of moving on I'll rename it postcode